RELEASE.md

```diff
@@ -1,34 +1,60 @@
 # Release Process
 
-## Requirements
+This documentation guides you on how to release a new version of the Codacy Chart.
 
-Please make sure you have these tools installed before starting this process:
+The release process consists of the following stages:
 
--   git
--   [ytool](https://github.com/codacy/ytool)
--   [helm >=3.x](https://helm.sh/docs/intro/install/)
+1.  **Deciding to release a new version**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=5647605)